lib/Service/CollectiveHelper.php

```diff
@@ -59,7 +62,8 @@ public function getCollectivesForUser(string $userId, bool $getLevel = true, boo
 				$userPageOrder = ($settings ? $settings->getSetting('page_order') : null) ?? Collective::defaultPageOrder;
 				$userShowRecentPages = ($settings ? $settings->getSetting('show_recent_pages') : null) ?? Collective::defaultShowRecentPages;
 			}
-			$collectiveInfos[] = new CollectiveInfo($c,
+			$collectiveInfos[$c->getId()] = new CollectiveInfo(
```

Comment:
Quickly checked and can reproduce the cypress failure locally when loading the collectives app.

It seems that this changed the API response from an array to an object, so it either needs to revert this and apply the id index elsewhere just for the widgets or make use of an array_values on the controller to avoid that the response format is changed.

Reply:
I'll revert and deal with it on the only consumer using that mapping.